recipe/meta.yaml

```diff
@@ -42,17 +42,21 @@ requirements:
     - glibmm
     - hdf5
     - icu
+    - libaec
     - libboost-devel
+    - libcurl
     - libiconv
     - liblapack
     - libxmlpp-4.0
     - libxml2
+    - openssl
     - sigcpp-3.0
     - sqlite
     - zlib
   run:
     - python
     - blis
+    - libgfortran5
```

Comment:
In this case, it's probably an issue with another package that is leaking this requirement somehow? At least, I don't see how cyclus could depend on libgfortran without using a fortran compiler? 🤔